src/bilder/images/edxapp/templates/forum.env

```diff
@@ -0,0 +1,25 @@
+{{ with secret "mongodb-mitxonline/creds/forum" }}
+export MONGOHQ_URL="mongodb://{{ .Data.username }}:{{ .Data.password }}@mongodb-master.service.consul:27017/forum"
+{{ end }}
+{{ with secret "secret-mitxonline/edx-forum" }}
+export API_KEY="{{ .Data.forum_api_key }}"
+{{ end }}
+export RBENV_ROOT="/edx/app/forum/.rbenv"
+export GEM_HOME="/edx/app/forum/.gem"
+export GEM_PATH="/edx/app/forum/.gem"
+export PATH="/edx/app/forum/cs_comments_service/bin:/edx/app/forum/.rbenv/bin:/edx/app/forum/.rbenv/shims:/edx/app/forum/.gem/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+export RACK_ENV="production"
+export SINATRA_ENV="production"
+export SEARCH_SERVER="http://localhost:9200"
```

Comment:
Is this needed since we have the one below it?

Reply:
Another copy/paste from the forum_env file. Not sure if it will break if we don't set it.